backend-shared/config/db/kubernetesresourcetodbresourcemapping_test.go

```diff
@@ -1,130 +1,49 @@
-package db
+package db_test
 
 import (
 	"context"
-	"strings"
-	"testing"
 
-	"github.com/stretchr/testify/assert"
-)
-
-func TestCreateKubernetesResourceToDBResourceMapping(t *testing.T) {
-	SetupforTestingDB(t)
-	defer TestTeardown(t)
-	dbq, err := NewUnsafePostgresDBQueries(true, true)
-	if !assert.NoError(t, err) {
-		return
-	}
-	defer dbq.CloseDatabase()
-
-	ctx := context.Background()
-
-	kubernetesToDBResourceMappingpost := KubernetesToDBResourceMapping{
-		KubernetesResourceType: "test-resource_1",
-		KubernetesResourceUID:  "test-resource_uid",
-		DBRelationType:         "test-relation_type",
-		DBRelationKey:          "test-relation_key",
-	}
-	err = dbq.CreateKubernetesResourceToDBResourceMapping(ctx, &kubernetesToDBResourceMappingpost)
-	assert.NoError(t, err)
-
-	kubernetesToDBResourceMappingget := KubernetesToDBResourceMapping{
-		KubernetesResourceType: "test-resource_1",
-		KubernetesResourceUID:  "test-resource_uid",
-		DBRelationType:         "test-relation_type",
-		DBRelationKey:          "test-relation_key",
-	}
-
-	err = dbq.GetDBResourceMappingForKubernetesResource(ctx, &kubernetesToDBResourceMappingget)
-	if !assert.NoError(t, err) {
-		return
-	}
-	assert.Equal(t, kubernetesToDBResourceMappingpost, kubernetesToDBResourceMappingget)
-
-}
-
-func TestGetDBResourceMappingForKubernetesResource(t *testing.T) {
-	SetupforTestingDB(t)
-	defer TestTeardown(t)
-	dbq, err := NewUnsafePostgresDBQueries(true, true)
-	if !assert.NoError(t, err) {
-		return
-	}
-	defer dbq.CloseDatabase()
-
-	ctx := context.Background()
-
-	kubernetesToDBResourceMappingpost := KubernetesToDBResourceMapping{
-		KubernetesResourceType: "test-resource_2",
-		KubernetesResourceUID:  "test-resource_uid",
-		DBRelationType:         "test-relation_type",
-		DBRelationKey:          "test-relation_key",
-	}
-	err = dbq.CreateKubernetesResourceToDBResourceMapping(ctx, &kubernetesToDBResourceMappingpost)
-	assert.NoError(t, err)
-
-	kubernetesToDBResourceMappingget := KubernetesToDBResourceMapping{
-		KubernetesResourceType: "test-resource_2",
-		KubernetesResourceUID:  "test-resource_uid",
-		DBRelationType:         "test-relation_type",
-		DBRelationKey:          "test-relation_key",
-	}
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
 
-	err = dbq.GetDBResourceMappingForKubernetesResource(ctx, &kubernetesToDBResourceMappingget)
-	if !assert.NoError(t, err) {
-		return
-	}
-	assert.Equal(t, kubernetesToDBResourceMappingpost, kubernetesToDBResourceMappingget)
-
-	kubernetesToDBResourceMappingNotExist := KubernetesToDBResourceMapping{
-		KubernetesResourceType: "test-resource_2_not_exist",
-		KubernetesResourceUID:  "test-resource_uid_not_exist",
-		DBRelationType:         "test-relation_type_not_exist",
-		DBRelationKey:          "test-relation_key_not_exist",
-	}
-	//check for inexistent primary key
```

Comment:
As mentioned above, I recommend preserving this 'inexistent primary key' test and moving it over to the Ginkgo version.